src/main/java/pl/kinga/leczenie/specyfik/Specyfik.java

```diff
@@ -0,0 +1,87 @@
+package pl.kinga.leczenie.specyfik;
+
+import pl.kinga.leczenie.Jednostka;
+import pl.kinga.leczenie.sposobleczenia.SposobLeczenia;
+
+import javax.persistence.*;
+
+@Entity
+public class Specyfik {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String nazwa;
+
+    @Enumerated(EnumType.STRING)
+    private Jednostka jednostka;
+    private double ilosc;
```

Comment:
bezpieczniej zawsze dawać Double zamiast double - wtedy jak ktoś nie uzupełni to będziesz miała wartość null (i informację, ze to pole nie zostało wypełnione), a nie defaultowe 0